global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "5.0",
```

Comment:
Puzzling that this was on 5.0 before - was it definitely supposed to be referring to .NET version?

Reply:
Ah, this seems to actually be the version that .Net CLI uses. I'll change it back

https://learn.microsoft.com/en-us/dotnet/core/tools/global-json

Reply:
I actually think we probably should update this tbh. Suspect it was missed when updating to .NET 6. Sorry fort the back-and-forth!

Reply:
No worries, I'll update it now :)